oap-server/pom.xml

```diff
@@ -83,7 +83,7 @@
         <curator-test.version>2.12.0</curator-test.version>
         <etcd4j.version>2.17.0</etcd4j.version>
         <etcd.version>v3.2.3</etcd.version>
-        <netty.version>4.0.27.Final</netty.version>
+        <netty.version>4.1.27.Final</netty.version>
```

Comment:
Why are you changing netty version? Something incompatible?

Reply:
Because etcd4j use 4.1.32 but grpc use 4.1.27. so i change the version, 
Keep them consistent. then need not add other netty jar to dist.